editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -502,7 +504,8 @@ describe('interactionStrategyChangeStacked', () => {
       interactionSession.interactionData.prevDrag = canvasPoint({ x: 30, y: 120 })
     }
     const editorStore = createEditorStore(interactionSession)
-    const actualResult = interactionStrategyChangeStacked(
+    editorStore.strategyState.currentStrategy = 'EMPTY_TEST_STRATEGY' as CanvasStrategyId
+    const actualResult = interactionUpdate(
```

Comment:
This looks to still be in a test block marked as `interactionStrategyChangeStacked`.